aspnetcore/host-and-deploy/health-checks.md

```diff
@@ -300,10 +300,34 @@ The `SampleHealthCheckWithDiConfig` and the Health check needs to be added to th
 
 :::code language="csharp" source="~/host-and-deploy/health-checks/samples/7.x/HealthChecksSample/Snippets/Program.cs" id="snippet_MapHealthChecksUsingDependencyInjection":::
 
+## UseHealthChecks vs. MapHealthChecks
+
+The `MapHealthChecks` method accepts arguments similar to `UseHealthChecks`. The advantage of using `MapHealthChecks` over `UseHealthChecks` is the ability to apply authorization and to have greater fine-grained control over the matching policy. The primary advantage of using `UseHealthChecks` over `MapHealthChecks` is terminating the middleware and selecting endpoints via port.
+
+<xref:Microsoft.AspNetCore.Builder.HealthCheckApplicationBuilderExtensions.UseHealthChecks%2A>:
+* Terminates the pipeline when a request matches the health check endpoint. [Short-circuiting](xref:fundamentals/middleware/index) is often desirable because it avoids unnecessary work, such as logging and other middleware.
+* Is primarily used for configuring the health check middleware in the pipeline.
+* Can select any endpoint on a port with a `null` or empty `PathString`, which can be useful for the following scenarios:
+  * Provides a default health check without specifying a path. Allows performing a health check on the root path or any request made to the specified port.
+  * Selecting any endpoint on a port.
+  * Provides a catch-all health check. Handles health checks for any path.
```

Comment:
These are all saying the same thing.

Reply:
> These are all saying the same thing.

@JamesNK  Can you make a suggestion of the lines to delete?